src/overlays/actors/ovl_En_Goroiwa/z_en_goroiwa.c

```diff
@@ -1,15 +1,40 @@
+/*
+ * File: z_en_goroiwa.c
+ * Overlay: ovl_En_Goroiwa
+ * Description: Rolling boulders
+ */
+
 #include "z_en_goroiwa.h"
 
+#include <vt.h>
+
 #define FLAGS 0x00000010
 
 #define THIS ((EnGoroiwa*)thisx)
 
+typedef bool (*EnGoroiwaUnkFunc1)(EnGoroiwa* this, GlobalContext* globalCtx);
+typedef void (*EnGoroiwaUnkFunc2)(EnGoroiwa* this);
+
 void EnGoroiwa_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnGoroiwa_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnGoroiwa_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnGoroiwa_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+bool func_80A4C814(EnGoroiwa* this, GlobalContext* globalCtx);
```

Comment:
prototypes for functions which are defined before being used such as `func_80A4C814` can be omitted